README.md

```diff
@@ -238,6 +251,63 @@ You can destroy everything using `terraform destroy`:
 terraform destroy
 ```
 
+#### Terraform ACI DevOps Agents - Assign identities
+
+This module allows to assign both system and user assigned managed identities to the containers:
+
+NB: managed identities for container groups have limitations. See <https://docs.microsoft.com/en-us/azure/container-instances/container-instances-virtual-network-concepts#other-limitations> and <https://docs.microsoft.com/en-us/azure/container-instances/container-instances-managed-identity#limitations>.
```

Comment:
Can you just specify in the readme that it will works only with Linux agent, due to the limitations, please? I prefer to have it written explicitly here.

Reply:
done in commit ce0e4e202f336a57322dd09f8e88b33a144ab2fa